labelbox/schema/task_queue.py

```diff
@@ -0,0 +1,28 @@
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+
+
+class TaskQueue(DbObject):
```

Comment:
Please add this class as an entry in `docs/source/index.rst` to be indexed for readthedocs.
More details https://labelbox.atlassian.net/wiki/spaces/AIENG/pages/1721139216/Release+a+new+Python+SDK+Version#Documentation-Updates